templates/base.html

```diff
@@ -51,13 +51,12 @@
 		</div><!-- /.container for main content -->
 
 		<footer class="footer">
-				<div class="container">
+				<div class="container">	
 					<a href="https://github.com/CodeCatz">
-					<img alt="CodeCatz" src="{% static 'CodeCatz.jpg' %}">
+						<img alt="CodeCatz" src="{% static 'CodeCatz.jpg' %}">
+						<p>GITHUB CodeCatz: Kittens learning to code like Catz</p>
 					</a>
-				</div>
+				</div>							
```

Comment:
Good job! New commits will automatically be added to the Pull request if you have one open :)